api/src/main/java/jakarta/enterprise/inject/spi/BeanContainer.java

```diff
@@ -251,4 +251,42 @@ public interface BeanContainer {
      * @since 2.0
      */
     Instance<Object> createInstance();
+
+    /**
+     * Returns true if a bean with given bean types and qualifiers would be assignable
+     * to an injection point with given required type and required qualifiers, false otherwise.
+     * In other words, this method provides access to the typesafe resolution rules - the process of matching a bean to an injection point.
+     * <p>
+     * Callers do not need to include implicit qualifiers such as {@code @Default} or {@code Any}.
```

Comment:
I'm no English expert, but "such as" implies that there may be more than just what you list, which AFAIK is not true. How about this?

```suggestion
Callers do not need to include implied qualifiers ({@code @Default}, {@code Any}).
```